app/views/user_case_infos/_form.html.erb

```diff
@@ -37,10 +42,20 @@
   <div class="field">
     <%= f.label :case_conclusion %><br>
     <%= f.text_area :case_conclusion, :title => 'Enter the Conclusion for Your Case', :class => 'tooltips'  %>
+    <select>
+      <option value="no papered">no papered</option>
+      <option value="dismissed">dismissed</option>
+      <option value="nolle prosequi">nolle prosequi</option>
+      <option value="guilty">guilty</option>
+    </select>
   </div>
   <div class="field">
     <%= f.label :is_case_pending %><br>
     <%= f.text_field :is_case_pending, :title => 'Is Your Case Still Pending', :class => 'tooltips'  %>
+    <form action="">
```

Comment:
This form tag, and the corresponding closing tag aren't needed (same on the edit page as well)